ui/app/components/clients/attribution.js

```diff
@@ -60,6 +57,11 @@ export default class Attribution extends Component {
     return attributionLegend;
   }
 
+  get formattedStartDate() {
```

Comment:
I accidentally removed this in [this PR](https://github.com/hashicorp/vault/blob/342b9c09bc85668a21de925065afb0bc56b793df/ui/app/components/clients/attribution.js#L214-L220](https://github.com/hashicorp/vault/pull/25733/files#diff-0a85892e181155b32233358b530f08a25f65ad94496d5c7961b5f463df634d73L54-L57) 🤦